axelor-account/src/main/java/com/axelor/apps/account/service/fixedasset/AbstractFixedAssetLineComputationServiceImpl.java

```diff
@@ -211,6 +210,18 @@ public Optional<FixedAssetLine> computeInitialPlannedFixedAssetLine(FixedAsset f
   }
 
   protected boolean isAlreadyDepreciated(FixedAsset fixedAsset) {
+    BigDecimal expectedDepreciation =
+        fixedAssetLineToolService.getCompanyScaledValue(
+            fixedAsset.getGrossValue(),
+            fixedAsset.getResidualValue(),
+            fixedAsset,
+            BigDecimal::subtract);
+
+    BigDecimal actualDepreciation = getAlreadyDepreciatedAmount(fixedAsset);
+
+    if (fixedAsset.getFixedAssetCategory().getIsProrataTemporis()) {
```

Comment:
Can you check that the fixed asset is not null and the category too to avoid NPE